types/react-native-video/index.d.ts

```diff
@@ -0,0 +1,77 @@
+// Type definitions for react-native-swiper 1.0
+// Project: https://github.com/react-native-community/react-native-video
+// Definitions by: HuHuanming <https://github.com/huhuanming>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import * as React from 'react';
+import {
+    ViewProperties
+} from 'react-native';
+
+export interface VideoProperties extends ViewProperties {
+    /* Native only */
+    src?: any;
```

Comment:
https://github.com/react-native-community/react-native-video/blob/master/Video.js

But it exists in the source.
```
Video.propTypes = {
  /* Native only */
  src: PropTypes.object,
  seek: PropTypes.number,
  fullscreen: PropTypes.bool,
  onVideoLoadStart: PropTypes.func,
  onVideoLoad: PropTypes.func,
  onVideoBuffer: PropTypes.func,
  onVideoError: PropTypes.func,
  onVideoProgress: PropTypes.func,
  onVideoSeek: PropTypes.func,
  onVideoEnd: PropTypes.func,
  onTimedMetadata: PropTypes.func,
  onVideoFullscreenPlayerWillPresent: PropTypes.func,
  onVideoFullscreenPlayerDidPresent: PropTypes.func,
  onVideoFullscreenPlayerWillDismiss: PropTypes.func,
  onVideoFullscreenPlayerDidDismiss: PropTypes.func,
```